pages/common/tbl.md

```diff
@@ -0,0 +1,24 @@
+# tbl
+
+> Format tables for troff.
+> More information: <https://manned.org/tbl>.
+
+- Format tables in troff:
+
+`cat {{path/to/file.troff}} | tbl`
+
+- Format tables in troff in "compatibility mode":
+
+`cat {{path/to/file.troff}} | tbl -C`
+
+- Format tables in troff from a file:
+
+`tbl {{path/to/file.troff}}`
+
+- Mark the beginning of a table ("Table Start"):
+
+`.TS`
+
+- Mark the end of the table ("Table End"):
+
+`.TE`
```

Comment:
I am not sure if we `should` or `should not` add these here ... 
lets see what others have to say :)

Reply:
Well ideally we would have an example, because that's what we do, but that's too long, as I already mentioned.

Reply:
I also think we can keep them. We document interactive shortcuts quite often (e.g. `vim`, `emacs`)